roles/libvirt_manager/README.md

```diff
@@ -46,7 +46,7 @@ cifmw_libvirt_manager_configuration:
   vms:
     type_name:  # (string, such as "compute", "controller"
       start: (boolean, toggle if the VM should be strated or not. Optional, defaults to true)
-      manage: (boolean, toogle if the VM should be added to the inventory. Managed VM's must be on the cifmw_libvirt_manager_pub_net network. Optional, defaults to true)
+      manage: (boolean, toggle if the VM should be added to the inventory. Managed VM's must be on the cifmw_libvirt_manager_pub_net network. Optional, defaults to true)
```

Comment:
that doesn't really belong to this patch I think :)

Reply:
correct, but documentation action was failing...